src/Renci.SshNet/Channels/ChannelDirectTcpip.cs

```diff
@@ -31,6 +33,7 @@ internal sealed class ChannelDirectTcpip : ClientChannel, IChannelDirectTcpip
         public ChannelDirectTcpip(ISession session, uint localChannelNumber, uint localWindowSize, uint localPacketSize)
             : base(session, localChannelNumber, localWindowSize, localPacketSize)
         {
+            _logger = SshNetLoggingConfiguration.LoggerFactory.CreateLogger<ChannelDirectTcpip>();
```

Comment:
There is a curious difference between the generic and non-generic CreateLogger, see https://github.com/dotnet/aspnetcore/pull/50184 and related links. Basically, the non-generic seems to be the way to go

Reply:
hm, feels like something .NET should fix instead but I would be fine with changing it. It would require disabling [CA2263](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2263) though. Do you want me change it?